### ISSUE TITLE
ENV in generate_centroids_solr may be misspelled RAILS_ENV

### ISSUE THREAD

**Issue by mberkowski** [BTAA-Geospatial-Data-Project/geoportal#186]
In the `generate_centroids_solr` task, `ENV` is used if `MY_ENV` is not defined, but this should maybe be `RAILS_ENV`?

Current `develop` deploy (rails_env=production) fails at the `generate_centroids_json` task because `CONFIG` has been defined mistakenly for the `development` env. A recent change points the development solr URL at a non-existing core, and the requisite environment variable `SOLR_URL` does not exist either. https://github.com/BTAA-Geospatial-Data-Project/geoportal/commit/2588582f61d4c024003662fdb0208ea6cb4400fe#diff-4babd2d6ff0bbdea92f6f8cba96dacbe

It seems like if the `generate_centroids_solr` task were changed to use `RAILS_ENV`,it would propagate `production` down through all the deploy stages correctly.

```
DEBUG [0dc22605] Command: cd /swadm/var/www/geoblacklight/releases/20190226171246 && bundle exec rake geoportal:generate_centroids_json RAILS_ENV=production
 DEBUG [b2752a9b] 	Configuration file: /swadm/var/www/geoblacklight/releases/20190226171246/config/jekyll.yml
 DEBUG [b2752a9b] 	       Jekyll Feed: Generating feed for posts
 DEBUG [b2752a9b] 	rake aborted!
URI::InvalidURIError: bad URI(is not URI?): "<%= ENV['SOLR_URL'] || \"http://127.0.0.1:8983/solr/blacklight-core\" %>/"
/swadm/var/www/geoblacklight/shared/bundle/ruby/2.6.0/gems/rsolr-2.2.1/lib/rsolr/client.rb:43:in `extract_url_from_options'
/swadm/var/www/geoblacklight/shared/bundle/ruby/2.6.0/gems/rsolr-2.2.1/lib/rsolr/client.rb:26:in `initialize'
/swadm/var/www/geoblacklight/shared/bundle/ruby/2.6.0/gems/rsolr-2.2.1/lib/rsolr.rb:22:in `new'
/swadm/var/www/geoblacklight/shared/bundle/ruby/2.6.0/gems/rsolr-2.2.1/lib/rsolr.rb:22:in `connect'
/swadm/var/www/geoblacklight/releases/20190226171246/lib/tasks/geoportal/generate_centroids_json.rake:7:in `block (2 levels) in <top (required)>'
/swadm/var/www/geoblacklight/shared/bundle/ruby/2.6.0/gems/rake-12.3.2/exe/rake:27:in `<top (required)>'
Tasks: TOP => geoportal:generate_centroids_json
(See full trace by running task with --trace)
```

**Comment by mberkowski** [comment]
Example - might need to change:

```
MY_ENV = ENV['RAILS_ENV'] || 'development'
```

**Comment by ewlarson** [comment]
Actually! This rake task is not necessary anymore. Karen is automatically adding centroids to our solr docs herself now.

The only centroid-related task that is necessary is the generate_centroids_json task that pre-populates the homepage map's clusters/centroids.

I'll remove the old task from the repo.

**Comment by mberkowski** [comment]
Ok if it gets removed and generate_centroids_json lives on, then the bit from the solr task that loads blacklight.yml into `CONFIG` still needs to be fixed and moved into generate_centroids_json. I think?

**Comment by ewlarson** [comment]
Yup. That's odd. Here's a better way to "do it" I think should work:
https://github.com/BTAA-Geospatial-Data-Project/geoportal/blob/feature/fullscreen-toggle/lib/tasks/geoportal/generate_centroids_json.rake#L7

That's a change I made while testing the application with a solrwrapper setup.

**Comment by mberkowski** [comment]
Fixed in f246f8a24d467ae908a9bc9d752e419d270fb1c9
